pkg/security/resolvers/netns/resolver.go

```diff
@@ -432,12 +436,14 @@ func (nr *Resolver) flushNetworkNamespace(netns *NetworkNamespace) {
 	// if we can, make sure the manager has a valid netlink socket to this handle before removing everything
 	handle, err := netns.getNamespaceHandleDup()
 	if err == nil {
-		defer handle.Close()
```

Comment:
where are we closing handle now ?

Reply:
I fixed it in my last two commits. I wrongly assumed that `netns.close()` would close the same handle but it was not the case.